app/controllers/public_pages_controller.rb

```diff
@@ -9,6 +9,11 @@ def announcements
     @announcements = Announcement.where(is_approved: true).published
   end
 
+  def announcements_json
+    announcements = Announcement.where(is_approved: true).published
+    render json: announcements
+  end
```

Comment:
To be more idiomatically Rails-y, you could roll this into the `announcements` method above. Here's a similar example elsewhere in this project: https://github.com/rubyforgood/mutual-aid/blob/9373a91680ab76603e706f4ea2eb22f3fe8cf936/app/controllers/contributions_controller.rb#L11-L14

And then Rails should pick the correct option if you explicitly tell it `<path>.html` versus `<path>.json` in your URL

Reply:
I had a feeling that's what was happening there, but I wasn't sure. Thanks!

Reply:
Would you like me to make that change before the merge, or should that be a separate pr?

Reply:
If you're ok making the change, would be best to do that before merging, IMO. Less to keep track of :)